src/CommunityToolkit.Maui.Core/Platform/StatusBar/StatusBar.android.cs

```diff
@@ -18,8 +19,9 @@ static void PlatformSetColor(Color color)
 		{
 			if (Activity.Window is not null)
 			{
-				Activity.Window.SetStatusBarColor(color.ToPlatform());
-				bool isColorTransparent = color.ToPlatform() == Android.Graphics.Color.Transparent;
+				PlatformColor platformColor = color.ToPlatform();
```

Comment:
```suggestion
				var platformColor = color.ToPlatform();
``` 

why not ` var` ?

Reply:
you are correct! we can use `var`, but I wanted to set a specific typed variable, I can change it, if needed 😃

Reply:
just out of curiosity, since, from our code style, we prefer the usage of `var`

Reply:
you are totally correct @pictos and I agree. I updated the code to follow the code style 😄